skrf/network.py

```diff
@@ -907,7 +904,11 @@ def __str__(self) -> str:
         if _z0.ndim < 2:
             z0 = _z0
         else:
-            z0 = self.z0[0, :]
+            if _z0.size > 0:
+                z0 = _z0[0, :]
+            else:
+                # empty frequency range
+                z0 = float('nan')
```

Comment:
Did the previous version was an issue in some situation? If yes, could you add a dedicated test?

Reply:
@jhillairet I added a test. The problem occurs in case one creates a network with an empty frequency range. That should not happen often in practice, but it can happen and the representation method of the Network object should work in that case.